src/renderer/components/drawer/drawer-item.tsx

```diff
@@ -1,25 +1,29 @@
 import "./drawer-item.scss";
 import React from "react";
-import { cssNames } from "../../utils";
+import { cssNames, displayBooleans } from "../../utils";
 
 export interface DrawerItemProps extends React.HTMLAttributes<any> {
   name: React.ReactNode;
   className?: string;
   title?: string;
   labelsOnly?: boolean;
   hidden?: boolean;
+  renderBoolean?: boolean; // show "true" or "false" for all of the children elements are "typeof boolean"
 }
 
 export class DrawerItem extends React.Component<DrawerItemProps> {
   render() {
-    const { name, title, labelsOnly, children, hidden, ...elemProps } = this.props
-    let { className } = this.props;
+    const { name, title, labelsOnly, children, hidden, className, renderBoolean, ...elemProps } = this.props
     if (hidden) return null
-    className = cssNames("DrawerItem", className, { labelsOnly });
+
+    const classNames = cssNames("DrawerItem", className, { labelsOnly });
+    const content = displayBooleans(renderBoolean, children)
```

Comment:
That is not good enough because if for some reason anyone uses the (imo unfortunately) ubiquitous pattern of `cond && expr` inside a drawer item, then instead of rendering nothing it would render a bunch of `false`'s.

Reply:
@Nokel81 I've checked example with PodDetails component in line https://github.com/lensapp/lens/blob/master/src/renderer/components/+workloads-pods/pod-details.tsx#L82 and there's no problem if keeping 
`const content = typeof children === "boolean" ? children.toString() : children`
inside `DrawerItem`. So double check if can we remove `displayBooleans` wrapper

Reply:
Line 82 is irreverent because it is not `typeof === "boolean"` but `typeof === "string"`. If that line happened to be `nodeName.length > 0 && ( ... )` then it wouldn't work.

Reply:
It actually works okay with `nodeName.length > 0 && ( ... )`:
![node name](https://user-images.githubusercontent.com/9607060/98208507-71145480-1f4e-11eb-9e34-44d5c82682db.png)

Reply:
It "works" in so far as if the LHS is truthy then the RHS is the result. However, if it is falsy then it would display `Node     false`